<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_sections</class>
 <widget class="QDialog" name="create_sections">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>create_sections</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="SelectGLGL">
      <property name="text">
       <string>Select Guide Line Group:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="GLGCB"/>
    </item>
    <item>
     <widget class="QLabel" name="SelectSurfacesL">
      <property name="text">
       <string>Select Surfaces:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="SelectSurfacesLW"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="CreateB">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CancelB">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
